lib/Elastica/Multi/MultiBuilder.php

```diff
@@ -38,16 +38,15 @@ private function buildResultSet(Response $childResponse, BaseSearch $search)
      */
     private function buildResultSets(Response $response, $searches)
     {
-        $resultSets = array();
-
         $data = $response->getData();
-        if (!isset($data['responses']) || !is_array($data['responses'])) {
-            return $resultSets;
+        if (empty($data['responses']) || !is_array($data['responses'])) {
```

Comment:
Not sure I get this as:
```
$data['responses'] = false;
empty($data['responses']) // true
!isset($data['responses']) // false
```

Even if his currently the same affect, the question is what our intention is: Do we want to check if the array entry is set or checking if it is empty? These are conceptually two different things and I think it is the first one we are looking for. So I would not change this line.